homeassistant/components/netatmo/light.py

```diff
@@ -134,3 +150,64 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     def async_update_callback(self) -> None:
         """Update the entity's state."""
         self._is_on = bool(self._camera.floodlight == "on")
+
+
+class NetatmoLight(NetatmoBase, LightEntity):
+    """Representation of a dimmable light by Legrand/BTicino."""
+
```

Comment:
```suggestion
_dimmer: NaModules.NLFN
```
I think you can do this to not need to do cast  on line 166

Reply:
I don't think this will work. But instead of using NLFN I rather should be using s.th. like `NaModules.Dimmer` (which I forgot to expose)

Reply:
An upstream PR is already submitted.